ext/tidy/tidy.c

```diff
@@ -1155,8 +1155,8 @@ PHP_FUNCTION(tidy_get_opt_doc)
 	opt = tidyGetOptionByName(obj->ptdoc->doc, optname);
 
 	if (!opt) {
-		php_error_docref(NULL, E_WARNING, "Unknown Tidy Configuration Option '%s'", optname);
-		RETURN_FALSE;
+		zend_argument_value_error(getThis() ? 1 : 2, "is an invalid configuration option");
+		RETURN_THROWS();
```

Comment:
This feels like one of the cases where showing the passed string in the error message might make sense.

Reply:
As I don't have any hard feelings against this, let's do it. But in the future (for PHP 8.1+) we should consolidate when to show these values, because this message will be an outlier from the rest now. :)